modules/ROOT/pages/engine-architecture-overview.adoc

```diff
@@ -334,7 +334,7 @@ Generic services  perform actions that are not related to BPM but are required f
 | Uses the Identity service to check user credentials
 |===
 
-=== Queriable logs
+=== Queryable logs
```

Comment:
⚠️ not sure we can change this, even if incorrect if this is the name that had been choosen in the code 😞